examples/hqca_circuit_osq.py

```diff
@@ -1,20 +1,23 @@
 from pathlib import Path
 from typing import Any, Dict, List
 
+from opensquirrel import CircuitBuilder
 from opensquirrel.squirrel_ir import Qubit
 
-from quantuminspire.sdk.models.circuit_v3 import CircuitV3
+# from quantuminspire.sdk.models.circuit_v3 import CircuitV3
```

Comment:
Why do we have a version to use the (low-level) API calls from OpenSquirrel? The SDK offers functionality via its models to wrap OpenSquirrel

Reply:
These are the scripts I made, but actually unnecessary because the qaoa and hqca present already used opensquirrel inside the Circuit object. Will eliminate these.